dts/bindings/rtc/adi,ds3231.yaml

```diff
@@ -0,0 +1,14 @@
+description: |
+  DS3231 as a RTC device
+
+compatible: "adi,ds3231"
+
+include:
+  - name: rtc-device.yaml
+  - name: i2c-device.yaml
+
+properties:
+  int1-gpios:
+    type: phandle-array
+    description: |
+      GPIO connected to the DS3231 INT1 interrupt output. This signal is open-drain, active low.
```

Comment:
DS3231 only has ~INT|SQW gpio output. Perhaps confusing to label this as INT1, suggesting that there's other INT(N) outputs?